TicketManagement/src/main/java/com/example/TicketManagement/TicketManagementApplication.java

```diff
@@ -0,0 +1,15 @@
+package com.example.TicketManagement;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+
+@SpringBootApplication
+@EnableJpaRepositories(basePackages = "com.example.TicketManagement.repository") // Specify the package where your repositories are located
```

Comment:
This annotation is not necessary as SpringBoot will enable it automatically

Reply:
Done